presto-spark-base/src/main/java/com/facebook/presto/spark/PrestoSparkTestingServiceWaitTimeMetrics.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.spark;
+
+import java.time.Duration;
+
+public class PrestoSparkTestingServiceWaitTimeMetrics
+        implements PrestoSparkServiceWaitTimeMetrics
+{
+    private static Duration waitTimeDuration = Duration.ofMillis(0);
```

Comment:
nit: It doesn't have to be static if you bind it in a singleton scope. Then you can provide a getter in the `TestSparkQueryRunner` that uses an injector to get the instance. That's what we usually do for other tests, e.g.: https://github.com/prestodb/presto/blob/55f99279442b165e3a4086a219ace7c94c74237e/presto-main/src/main/java/com/facebook/presto/server/testing/TestingPrestoServer.java#L543